core/modules/simpletest/tests/ajax_forms_test.module

```diff
@@ -254,6 +254,15 @@ function ajax_forms_test_ajax_commands_form($form, &$form_state) {
     ),
   );
 
+  // Shows the Ajax 'add_css' command.
+  $form['add_css_command_example'] = array(
+    '#type' => 'submit',
+    '#value' => t("AJAX 'add_css' command"),
```

Comment:
I think if `add_css` (in the #value) is changed to `addCss` here it will fix the tests. The comment should be updated too, but it's not important.